cirq/google/calibration/phased_fsim.py

```diff
@@ -155,8 +167,35 @@ def get_parameters(self, a: Qid, b: Qid) -> Optional['PhasedFSimCharacterization
         else:
             return None
 
+    @classmethod
+    def _from_json_dict_(
+        cls,
+        parameters: List[Dict[str, Any]],
+        gate: Gate,
+        **kwargs,
+    ) -> 'PhasedFSimCalibrationResult':
+        """Magic method for the JSON serialization protocol.
+
+        Converts serialized dictionary into a dict suitable for
+        class instantiation."""
+        instantiation_dict = {}
+        for entry in parameters:
+            key = (entry['qubit_a'], entry['qubit_b'])
+            instantiation_dict[key] = entry['characterization']
+        return cls(instantiation_dict, gate)
+
+    def _json_dict_(self) -> Dict[str, Any]:
+        """Magic method for the JSON serialization protocol."""
+        return {
+            'cirq_type': 'PhasedFSimCalibrationResult',
+            'gate': self.gate,
+            'parameters': [
+                {'qubit_a': key[0], 'qubit_b': key[1], 'characterization': self.parameters[key]}
```

Comment:
you could also simplify to 
(q0, q1, parameters for ((q0, q1), parameters) in self.parameters.items()]
and deserialization would be:
instantaition_dict = {(q0, q1): params) for q0, q1, params in parameters]

up to you

Reply:
Changed.